scripts/prepForBuild.js

```diff
@@ -9,7 +9,7 @@ let copyFiles = async function() {
     console.log(`${bundle} was copied to ./build dir`);
   });
 
-  fs.copyFile(`./demo/css/style.css`, `./build/css/style.css`, (err) => {
+  fs.copyFile(`./demo/style.css`, `./build/style.css`, (err) => {
```

Comment:
Shouldn't this account for:

```suggestion
  fs.copyFile(`./demo/sass/style.scss`, `./build/style.scss`, (err) => {
```

Reply:
The Sass directory has been removed from the demo directory.